doc/orbits.xml

```diff
@@ -13,11 +13,11 @@
   <Attr Name="DigraphGroup" Arg="digraph"/>
   <Returns>A permutation group.</Returns>
   <Description>
-    If <A>digraph</A> was created knowing a subgroup of its automorphism group,
-    then this group is stored in the attribute <C>DigraphGroup</C>. If
-    <A>digraph</A> is not created knowing a subgroup of its automorphism group,
-    then <C>DigraphGroup</C> returns the entire automorphism group of
-    <A>digraph</A>.
+    If <A>digraph</A> was immutable and created knowing a subgroup of
+    its automorphism group, then this group is stored in the attribute
+    <C>DigraphGroup</C>. If <A>digraph</A> is mutable, or was not created
+    knowing a subgroup of its automorphism group, then <C>DigraphGroup</C>
+    returns the entire automorphism group of <A>digraph</A>.
```

Comment:
Add "Note that if digraph is mutable, then the automorphism group is recomputed every time this function is called."